pvesubscription: add page (#23148)
tldr-pages/tldr: pages/linux/pvesubscription.md

# pvesubscription

> Manage Proxmox subscription keys.
> More information: <https://pve.proxmox.com/pve-docs/pvesubscription.1.html>.

- Display subscription information:

`pvesubscription {{[g|get]}}`

> - Set subscription key:

Reviewer: ```suggestion
- Set the subscription key:
```

`pvesubscription set {{subscription_key}}`

- Update subscription information:

`pvesubscription {{[u|update]}}`

> - Delete a subscription key of the current node:

Reviewer: ```suggestion
- Delete the subscription key from the current node:
```
